frontend/src/features/public-form/components/FormPaymentPage/stripe/StripeReceiptContainer.tsx

```diff
@@ -87,11 +87,15 @@ export const StripeReceiptContainer = ({
           paymentDate={paymentReceiptStatus.paymentDate}
         />
       </PaymentStack>
-      <PaymentStack>
-        {!isFeedbackSubmitted && (
-          <FeedbackBlock onSubmit={handleSubmitFeedback} />
-        )}
-      </PaymentStack>
+      {
```

Comment:
Seems to have an extra set of `{}` braces here, is it intended?

Reply:
ahh... I think I was still playing around then when I added it but will remove it!